hack/testing/assertions

```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
-source "$(dirname $0)/utils"
+source "$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )/utils"
```

Comment:
is it really necessary to `cd` as part of this `source` statement?

Reply:
in general, yes, if you want the "real" absolute path